WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -17,8 +17,11 @@ class CompetitionsController < ApplicationController
     :admin_edit,
   ]
 
-  private def competition_from_params
-    Competition.find(params[:competition_id] || params[:id]).tap do |competition|
+  private def competition_from_params(include_events: false, include_rounds: false)
```

Comment:
Yes, I saw that and definitely prefer your solution! I can definitely make that change.